### DISCUSSION TITLE
Wildcards in paths and filtering

### DISCUSSION THREAD

**Discussion by robflate** [garethgeorge/backrest#504] [Q&A]
A few questions in general about wildcards and filtering;

- Is it possible to use wildcards in paths like `/userdata/data/**`?
- Can I use `--include` flags with Backrest? I seem to get errors.
- What is the best workflow for using Backrest to do things like;

Exclude everything except .txt files?

`restic backup /repos/backup --exclude "/userdata/data/**" --include "/userdata/data/**/*.txt"`

Exclude contents but still backup folder?

`restic backup /repos/backup --exclude "/userdata/data/**" --include "/userdata/data/"`

Any guidance appreciated. Thanks.

**Comment by scialo** [comment]
use --files-from files_to_backup

https://restic.readthedocs.io/en/stable/040_backup.html#including-files
